crates/chronicle/src/bootstrap/mod.rs

```diff
@@ -333,9 +313,11 @@ where
             }
         }
 
-        if jwks_uri.is_none() && id_pointer.is_none() {
+        // This may not be the ideal place for this, but we need it for development
+        // as we default to an anonymous user when we serve graphql
+        if jwks_uri.is_none() {
             DirectoryStoredKeys::new(config.secrets.path)?
-                .generate_agent(&AgentId::from_external_id("Anonymous"))?;
+                .generate_agent(&AgentId::from_external_id("anonymous"))?;
```

Comment:
I've been curious if identities generated like this are materially distinct from `AuthId::anonymous()`.

Reply:
This generates a user key.